src/with-props.js

```diff
@@ -4,15 +4,15 @@ import type {
   HasConstructor,
```

Comment:
Only prettier.

Reply:
we need to automatize prettier stuff via lint-staged